justfile

```diff
@@ -17,3 +20,23 @@ coverage *FLAGS:
   uv run coverage run -m pytest tests -m "not integration" {{FLAGS}}
   uv run coverage report
   uv run coverage lcov -o lcov.info
+
+ai-exchange-version:
+  curl -s https://pypi.org/pypi/ai-exchange/json | jq -r .info.version
+
+# bump project version, push, create pr
+release version:
+  uvx --from=toml-cli toml set --toml-path=pyproject.toml project.version {{version}}
+  ai_exchange_version=$(just ai-exchange-version) && sed -i '' 's/ai-exchange>=.*/ai-exchange>='"${ai_exchange_version}"'\",/' pyproject.toml
+  git co -b release-version-{{version}}
+  git add .
```

Comment:
We'll want to be more specific here - too easy to have this accidentally commit something unintended. Just pyproject.toml should be enough?